test/Batch/CHANGED_SOURCES.py

```diff
@@ -48,8 +48,11 @@
 sys.exit(0)
 """)
 
+# Disable IMPLICIT_COMMAND_DEPENDENCIES because otherwise it renders them less
```

Comment:
Can you explain? Not sure I understand.

Reply:
Sorry this is a remnant of my change previously switching the default behavior. This can be reverted now that the default behavior is unchanged. I'll make a pass through the diff reverting unnecessary changes.